laws/src/main/scala/org/http4s/laws/discipline/arbitrary.scala

```diff
@@ -1105,7 +1105,10 @@ private[discipline] trait ArbitraryInstancesBinCompat0 extends ArbitraryInstance
     for {
       timeout <- Gen.option(Gen.chooseNum(0L, Long.MaxValue))
       max <- Gen.option(Gen.chooseNum(0L, Long.MaxValue))
-      l <- Gen.listOf(genExtension)
+      l <- Gen.listOf(genExtension).filterNot { ext =>
+        val extNames = ext.map(_._1)
+        extNames.contains("timeout") || extNames.contains("max")
+      }
```

Comment:
Should this filtering happen in the definition `genExtension` itself, if these are invalid extension names?

Reply:
Sure, it will look prettier